e2e/go.mod

```diff
@@ -3,11 +3,11 @@ module github.com/ssvlabs/ssv/e2e
 go 1.20
 
 require (
-	github.com/attestantio/go-eth2-client v0.19.11-0.20240129201044-9d799aaab2bd
+	github.com/attestantio/go-eth2-client v0.21.1-0.20240325145558-858fa8a893fa
```

Comment:
what is this version? I thought we said lets update to [v0.21.3](https://github.com/attestantio/go-eth2-client/releases/tag/v0.21.3)
this commit is not an official tag - [858fa8a893fa](https://github.com/attestantio/go-eth2-client/commit/858fa8a893fa)